data-raw/process_data_set_danica.R

```diff
@@ -0,0 +1,24 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+historical_RAPTOR_by_player <- read_csv("data-raw/nba-raptor/historical_RAPTOR_by_player.csv") 
+
+modern_RAPTOR_by_player <- read_csv("data-raw/nba-raptor/modern_RAPTOR_by_player.csv") 
```

Comment:
Instead of having `historical_raptor_by_player` and `modern_raptor_by_player` as two separate data frames, let's have a single data frame called `raptor_by_player` where the first variable is `era` is a `factor` equal to `historical` or `modern`. That way people can easily compare values for both eras.

First, note 6 variables are included in `modern_raptor_by_player` but not in `historical_raptor_by_player` (most likely due to advancements in scorekeeping)
```
> setdiff(names(modern_raptor_by_player), names(historical_raptor_by_player))
[1] "raptor_box_offense"   "raptor_box_defense"   "raptor_box_total"    
[4] "raptor_onoff_offense" "raptor_onoff_defense" "raptor_onoff_total" 
```
Create vectors in `historical_raptor_by_player` with these names and set the value to `NA`. Indicate in the help file that such values are missing for when `era` equals `historical`

Second, make sure that all 22 variables (which includes the new `era` variable) are listed in the same order and have the same type, then use `bind_rows()` to merge them together.